holoviews/plotting/mpl/tabular.py

```diff
@@ -117,7 +115,7 @@ def initialize_plot(self, ranges=None):
             for col in range(element.cols):
                 if summarize and row > half_rows:
                     adjusted_row = (element.rows - self.max_rows + row)
-                cell_value = self.cell_values[self.keys[-1]][(row, col)]
+                cell_value = element.pprint_cell(row, col)
```

Comment:
Shouldn't both of these calls also pass the return value through the ``pprint_value`` method on TablePlot, I'm not sure why both the element and the plot need a pprint method but I assume there was a reason.